src/runtimes/node/bundlers/zisi/src_files.ts

```diff
@@ -87,6 +88,19 @@ const getDependencies = async function ({
   }
 }
 
+const paperwork = async (path: string) => {
```

Comment:
Can we add a comment explaining that we're temporarily adding a step between us and precinct because of a specific issue?

Reply:
done in https://github.com/netlify/zip-it-and-ship-it/pull/1069/commits/c75179fea6142e65f0facd3174625829d5a66408